src/apps/monitoring/latencetech/restapi/mode/throughput.pm

```diff
@@ -0,0 +1,155 @@
+#
+# Copyright 2025 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package apps::monitoring::latencetech::restapi::mode::throughput;
+
+use base qw(centreon::plugins::templates::counter);
+
+use strict;
+use warnings;
+
+sub set_counters {
+    my ($self, %options) = @_;
+
+    $self->{maps_counters_type} = [
+        { name => 'global', type => 0, cb_prefix_output => 'prefix_output' }
+    ];
+
+    $self->{maps_counters}->{global} = [
+        { label => 'lifbe-download', nlabel => 'lifbe.download.bandwidth.mbps', set => {
+                key_values => [ { name => 'lifbeDownload' }, { name => 'display' } ],
+                output_template => 'LIFBE Download: %.2fmbps',
+                perfdatas => [
+                    { value => 'lifbeDownload', template => '%.2f',
+                      min => 0, unit => 'mbps', label_extra_instance => 1, instance_use => 'display' },
```

Comment:
The value should be in `bps` in the perfdata and converted to human only in the text output.